.circleci/real_config.yml

```diff
@@ -5479,6 +5560,65 @@ workflows:
             - github-read
             - dev-ci-cluster-default-user-credentials
 
+  release-dryrun:
+    jobs:
+      - build-helm:
+          filters: *release-dryrun
```

Comment:
okay, i see you're using those anchors from before. i still like giving regex patterns names if possible, with a comment if you want

Reply:
actually, instead of giving each job a filter, shouldn't the workflow itself just be conditional?
https://support.circleci.com/hc/en-us/articles/360043638052-Conditional-steps-in-jobs-and-conditional-workflows

Reply:
Filters tend to be used more for only running certain things based on git evenets. That being said, I could definitely factor out those filters. I was attempting to make the `release-dryrun` workflow as similar as possible to the `release` workflow, but a single workflow filter is provably equivalent to a non-filtered workflow with the same filter for each job.

Reply:
yeah, no need to fix if it's copypasta. similarity is good. but maybe we could consider fixing both workflows later